src/seedu/addressbook/commands/FindCommand.java

```diff
@@ -57,4 +54,19 @@ public CommandResult execute() {
         return matchedPersons;
     }
 
+    /**
+     * convert the keywords set to string array, iterate the array to convert the string to lower case and convert back to set again
+     */
+    private void changeKeywordToLowerCaseInSet()
+    {
+        String[] keywordArray = this.keywords.toArray(new String[keywords.size()]);
+
+        for(int i = 0; i< keywordArray.length; i++)
```

Comment:
Likewise for keywords like `if` `while` `for`, please refer to the coding standards for the module.